contracts/SimpleAssetsExchange.sol

```diff
@@ -1,24 +1,45 @@
 // SPDX-License-Identifier: UNLICENSED
 pragma solidity ^0.8.2;
 
+import "@pangolindex/exchange-contracts/contracts/pangolin-periphery/PangolinRouter.sol";
+import "@openzeppelin/contracts-ethereum-package/contracts/token/ERC20/IERC20.sol";
 import "@openzeppelin/contracts/access/Ownable.sol";
 import "./IPriceProvider.sol";
 import "./IAssetsExchange.sol";
 
 /**
- * @title SimpleAssetsExchange
+ * @title PangolinAssetsExchange
  * @dev Contract allows user to invest into an asset
- * It is a simple implementation that could be replace by a DEX or synthetic asset provider
+ * This implementation uses the Pangolin DEX
  */
-contract SimpleAssetsExchange is Ownable, IAssetsExchange {
-
+contract PangolinAssetsExchange is Ownable, IAssetsExchange {
   mapping(address => mapping(bytes32 =>uint256)) balance;
```

Comment:
The Pangolin DEX contract will act as a proxy to pangolin exchange - we don't need to keep tokens in this contract